stripe.go

```diff
@@ -696,10 +696,6 @@ func (s *BackendImplementation) ResponseToError(res *http.Response, resBody []by
 	switch raw.Error.Type {
 	case ErrorTypeAPI:
 		typedError = &APIError{stripeErr: raw.Error}
-	case ErrorTypeAPIConnection:
-		typedError = &APIConnectionError{stripeErr: raw.Error}
```

Comment:
Can we delete the underlying structs (eg. `APIConnectionError`)? Are they used by anything else?